controller/loadBasicPokemonDetails.js

```diff
@@ -7,20 +7,29 @@ let fs = require('fs'),
 function loadBasicPokemonDetails() {
     console.log('Loading Basic Pokemon Details to MongoDB');
 
-    let data = JSON.stringify(pokemonListPath, function (key, value) {
-        var pokemonID, pokemonName;
-        if (key != 'name' && typeof value.name !== "undefined") {
+    let data = JSON.stringify(pokemonListPath, function(key,value) {
+        var pokemonID, pokemonName, male, female;
+        if(key != 'name' && typeof value.name !== "undefined") {
             pokemonID = key;
             pokemonName = value.name;
-
+            male = value.gender.male;
+            female = value.gender.female;
+            /*if (female) {
+                gender = female;
+            } else {
+                gender = male;
+            }*/
+            //gender = value.gender.male;
             //add pokemonID and pokemonName to collection
-            basicPokemonDetail.create({pokemonID: pokemonID, name: pokemonName}, function (err, post) {
-                if (err) {
+            basicPokemonDetail.create({pokemonID: pokemonID, name: pokemonName, gender: {male:male,female:female}}, function(err, post){
```

Comment:
Why would you store two keys for a binary attribuite? If someone is female she is not male. This is against NF (especially in NRDBS) and redundant :)
Also: what are you exactly storing? A string? You only need a boolean (which takes up much less space in memory)